web/index.html

```diff
@@ -22,35 +23,59 @@
     // Version injected by flutter build, do not touch.
     var serviceWorkerVersion = null;
   </script>
-  <!-- Flutter init JS code -->
-  <script src="flutter.js" defer></script>
   <style>
     /* unminify.com */
     body{inset:0;overflow:hidden;margin:0;padding:0;position:fixed}
     #loading{align-items:center;background-color:#354046;display:flex;flex-direction:column;justify-content:center;height:100%;width:100%}
-    #loading h1{color:#f2f2f2;font-family:'Open Sans',sans-serif;font-size:30px;font-weight:300;line-height:32px;margin-top:20px;text-align:center}
+    #loading h1{color:#f2f2f2;font-family:'Open Sans',sans-serif;font-size:46px;font-weight:300;line-height:32px;margin-top:20px;text-align:center}
+    button{display:inline-block;font-size:42px;padding:.4em .7em;color:#FFF;background-color:#4bc0a9;text-align:center;text-shadow: 2px 2px 3px #354046}
```

Comment:
Adding `cursor: pointer`  to `button` would make sense here.

Reply:
Go for it: 
```suggestion
    button{display:inline-block;font-size:42px;padding:.4em .7em;color:#FFF;background-color:#4bc0a9;text-align:center;text-shadow: 2px 2px 3px #354046; cursor: pointer}
```

But remember: we don't care that much about the `Desktop` use case. 
And cursors are meaningless on `Mobile` ... 
It doesn't _detract_ from the code. So add it.

Reply:
Since all the changes of the PR are pertaining to the `web` version of the app, of course it's meaningless on mobile. It's just a detail, you can happily ignore. The rest is top-notch 👍